content/influxdb/v2.1/reference/internals/runtime.md

```diff
@@ -0,0 +1,343 @@
+---
+title: InfluxDB runtime
+description: >
+  Learn how to collect Go runtime profiling and tracing information to help with InfluxDB performance analysis and debugging.
+menu:
+  influxdb_2_1_ref:
+    name: Runtime
+    parent: InfluxDB internals
+weight: 103
+influxdb/v2.1/tags: [go, internals, performance]
+---
+
+InfluxDB provides Go runtime profiles, trace, and other information
+useful for analyzing and debugging the server runtime execution.
+
+- [Analyze Go runtime profiles](#analyze-go-runtime-profiles)
+- [Analyze the Go runtime trace](#analyze-the-go-runtime-trace)
+- [View the command line that invoked InfluxDB](#view-the-command-line-that-invoked-influxdb)
+
+## Analyze Go runtime profiles
```

Comment:
Let's split conceptual content into "Overview of Go runtime profiles" and follow with task-based "Analyze..."
More [detail ](https://developers.google.com/style/headings?hl=en#heading-and-title-text)